internal/message/repository.go

```diff
@@ -44,10 +62,29 @@ func (repo *messageRepository) SendMessage(ctx context.Context, toPersonID strin
 	from := mail.NewEmail("JobsGoWhere Message", "no-reply@jobsgowhere.com")
 	to := mail.NewEmail(toPersonName, toPerson.Email)
 
-	plainTextContent := senderName + "has sent a message" + body
-	htmlContent := "<strong>" + senderName + "</strong> has sent a message, <br>" + body
+	emailContent := EmailContent{
+		LogoURL:          os.Getenv("LOGO_URL"),
+		ReceiverImgURL:   toPerson.AvatarURL.String,
+		ReceiverName:     toPersonName,
+		ReceiverPosition: toPerson.R.PersonProfiles[0].ProfileType.String,
```

Comment:
@subhransu Thanks for helping to generate that screenshot! I'm supposed to inject it in using ReceiverPosition, ReceiverCompany and ReceiverHeadline but I'm not sure if this is the correct way to do it. Could you confirm?

Reply:
This is correct, I might have the data missing. Let me verify.